profiles/package.mask

```diff
@@ -32,6 +32,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Wolfgang E. Sanyer <WolfgangESanyer@Gmail.com> (2021-03-11)
```

Comment:
The mask claims it masks deprecated package. But it also masks last available version of softwware maintained by someone else. You will need to sort out the plan with them first.